### PR TITLE
[Nokia][DeviceData] Update the Nokia platform IXR-7250E device data

### PR DESCRIPTION
**Pull request by mlok-nokia**
#### Why I did it
Update the platform_reboot of Nokia Platform IXR-7250E-36x400G to displays the correct reboot-cause history when reboot from supervisor card. 

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Modify the platform_reboot script to copy the correct reboo-cause.txt file from NDK to the /host/reboot-cause directory at the down cycle when the reboot is issued from Supervisor (for both reboot  right after install a new image and normal reboot)

#### How to verify it
1)  Execute a CLI command "sudo reboot" on the Supervisor
2)  After system is up, on the Linecard, execute the CLI show reboot-cause history
```
Name                 Cause                                                                           Time                             User    Comment
-------------------  ------------------------------------------------------------------------------  -------------------------------  ------  ---------
2023_08_03_16_41_30  reboot from Supervisor                                                          Thu 03 Aug 2023 04:39:48 PM UTC          N/A
2023_08_03_00_42_16  reboot from Supervisor                                                          Thu 03 Aug 2023 12:39:58 AM UTC  admin   N/A
2023_08_02_04_04_58  reboot                                                                          Wed 02 Aug 2023 04:02:56 AM UTC  admin   N/A
2023_08_02_03_20_21  reboot                                                                          Wed 02 Aug 2023 03:18:18 AM UTC  admin   N/A
2023_08_01_16_14_57  reboot                                                                          Tue 01 Aug 2023 02:49:45 PM UTC  admin   N/A
2023_08_01_14_46_45  reboot                                                                          Tue 01 Aug 2023 02:44:43 PM UTC  admin   N/A

```

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205
- [x] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)
branch 202205 latest

- [ ] 
- [ ]

### CODE REVIEW

**Comment by mlok-nokia** [comment]
@judyjoseph  This PR is to correct to display the reboot cause when reboot from Supervisor card For Nokia platform.  Please review it. Thanks

File: device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_reboot [hunk 1]
```diff
@@ -1,31 +1,30 @@
...
-        if [ -f $REBOOT_CAUSE_FILE ]; then
-            t1=`sudo grep "User: ," $REBOOT_CAUSE_FILE`
-            if [ ! -z "$t1" ]; then
-                echo $t1 | sed 's/reboot/reboot from Supervisor/g' | sed 's/User: /User: admin/g' > $TMP_REBOOT_CAUSE_FILE
-                cp $TMP_REBOOT_CAUSE_FILE $REBOOT_CAUSE_FILE
-            fi
+    DEVICE_REBOOT_CAUSE_FILE=/etc/opt/srlinux/reboot-cause.txt
+    if [ -e  $DEVICE_MGR_REBOOT_FILE ]; then
+        if [ -e $DEVICE_REBOOT_CAUSE_FILE ]; then
+            cp -f $DEVICE_REBOOT_CAUSE_FILE $REBOOT_CAUSE_FILE
```
Comment (judyjoseph) [inline device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_reboot:10]: Does this change need a new NDK for this change ?

Comment (judyjoseph) [reply device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/platform_reboot:10]: I see the ndk already merged in https://github.com/Azure/sonic-buildimage-msft/pull/77, will go ahead and merge + mark for 202205

**Review by judyjoseph — APPROVED** [review]

**Review by judyjoseph — APPROVED** [review]

**Comment by mlok-nokia** [comment]
@judyjoseph @gechiang   Could you please cherry-pick this PR to 202205 branch?  Thanks

**Comment by gechiang** [comment]
@yxieca , @StormLiangMS , This Bug fix PR is backport needed for the corresponding branches.  This is very specific to the Nokia IXR-7250E platform only. should not impact anything else.
Thanks!

**Pull request merged**